dxr/plugins/python/tests/test_modules_packages/makefile

```diff
@@ -0,0 +1,13 @@
+all:
```

Comment:
You probably will actually want to rebase this PR, since all this boilerplate stuff can now go byebye.